src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -495,3 +495,29 @@
     text: az backup vault show --name MyRecoveryServicesVault --resource-group MyResourceGroup
     crafted: true
 """
+helps['backup vault update'] = """
+type: command
+short-summary: Updates an existing Recovery Services vault.
+examples:
+  - name: Updates an existing Recovery services vault. (autogenerated)
+    text: az backup vault update --identity-type None --resource-group MyResourceGroup --vault-name MyVault
```

Comment:
May I ask can the users not pass in the parameter  `-identity-type` instead of passing None?

Reply:
@zhoxing-ms user can pass None to remove all the identities at once if he wish. He can also pass SystemAssigned or UserAssigned as --identity-type for addition of identities

Reply:
If you wish we can disallow passing the 'None' value to the identity-type. In our case 'None' is a value user passes to remove all identities in a vault.

Reply:
Got it, thanks~